internal/db/schema/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql

```diff
@@ -113,7 +292,11 @@ begin;
        and c.relname operator(pg_catalog.~) '^(.+_deleted)$' collate pg_catalog.default
        and pg_catalog.pg_table_is_visible(c.oid);
   $$ language sql;
+  comment on function get_deletion_tables is
+    'get_deletion_tables returns a set containing all the deleted table names by looking for '
+    'all tables that end in _deleted.';
 
+  -- Assign a "before delete" trigger on all deleted tables to run insert_deleted_id
   do $$
```

Comment:
Why is this a `before delete` trigger and not an `after delete` trigger?

Reply:
> No particular reason outside of postgres suggesting `before` is more consistent, can switch it to an `after`

Where was this suggestion? Can you share a link?

Reply:
[At the bottom in compatibility](https://www.postgresql.org/docs/current/sql-createtrigger.html) - I may have misunderstood this though, after reading it again

Reply:
I changed this to `after delete`.